src/main/java/com/teamwizardry/refraction/common/effect/EffectBurn.java

```diff
@@ -68,13 +73,17 @@ public void runEntity(World world, Entity entity, int potency) {
 		if (entity instanceof EntityItem) {
 			EntityItem item = (EntityItem) entity;
 			ItemStack stack = item.getItem();
-			if (FurnaceRecipes.instance().getSmeltingResult(stack) != ItemStack.EMPTY) {
-				if (ThreadLocalRandom.current().nextInt(100) == 0) {
```

Comment:
any reason you dropped this line?

Reply:
Actually it is quite tricky to trigger smelting, for balance consideration I removed this line. Maybe add a config option?

Reply:
Yea config would be the best approach. Do you want to add it to your PR, or shall i do it afterwards? :)